recipe/meta.yaml

```diff
@@ -135,21 +133,15 @@ outputs:
     script: repack.bat  # [win]
     number: {{ mkl_buildnum|int + dstbuildnum|int }}
     requirements:
-      {% if win or 'conda-forge' not in my_channel_targets %}
-      host:
-        - intel-openmp {{ openmp_version.split('.')[0] }}.*
-        - tbb {{ tbb_version.split('.')[0] }}.*
-      run:
-        - intel-openmp {{ openmp_version.split('.')[0] }}.*
-        - tbb {{ tbb_version.split('.')[0] }}.*
-      {% else %}
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ c_stdlib }}_{{ target_platform }} >={{ c_stdlib_sycl_version }}  # [linux]
+        - {{ stdlib('c') }}  # [not linux]
+        - {{ openmp_pkg }}
+        {{ openmp_mutex }}
```

Comment:
Also, this went wrong; needs to have a leading `- `

Reply:
Nevermind, I just saw that this mutex may be missing, in which case this makes sense.